### DISCUSSION TITLE
Version 4.2.2 - Logo remplacé par une roue crantée

### DISCUSSION THREAD

**Discussion by CHRBT22** [JeromeDevome/GRR#322] [General]
Bonjour,
J'ai basculé de la 4.2.0 à la 4.2.2 et depuis le logo est systématiquement remplacé par une roue crantée "verte" :
![image](https://github.com/JeromeDevome/GRR/assets/155730470/4e1767af-75d0-4351-b442-0a87e4665c64)
Une idée pour retrouver le comportement initial ?
Merci par avance,
Christophe

**Comment by JeromeDevome** [comment]
Bonjour, je pense que votre dossier image dans personnalisation ne doit pas avoir les bon droits pour écraser celle-ci.

**Reply by CHRBT22** [reply]
Bonjour,
Merci pour ce retour rapide, cependant les droits UGO me semblent corrects : 775 pour www-data:root sur les dossiers images et logos

**Reply by JeromeDevome** [reply]
Et si vous supprimer l'image dans le fichier logo, vous pouvez importer la nouvelle ?

**Reply by CHRBT22** [reply]
J'ai supprimé les 2 fichiers img_logo.jpg et img_logo.png présents dans le dossier personnalisation/images/logos.
J'importe à nouveau mon logo PNG via l'interface web et la roue crantée apparait.
Le fichier img_logo.png qui se retrouve dans le dossier personnalisation/images/logos est bien le fichier que j'ai importé.

**Reply by JeromeDevome** [reply]
Vous avez essayé en navigation privé ? Problème de cache peut-être.

**Reply by CHRBT22** [reply]
Le NOOB...
Effectivement c'est OK, pb de cache...
Désolé pour le dérangement et merci pour le temps pris :)
